### ISSUE TITLE
Import ICCU 2. Paket

### ISSUE THREAD

**Issue by jenniferward** [rism-international/tasks#1016]
Für das Import vom 2. ICCU-Paket wurden folgende Felder überarbeitet: 260 a, b, c, 300 a, c, 593, 592
Dabei wurden Bemerkungen ergänzt. Damit sie nicht mit den schon vorhandenen Bemerkungen in Konflikt geraten, habe ich sie in Feld 999 gespeichert.

**Für die Datei in MARCXML-Format (Mail folgt), 16.238 Titel**
1. Folgende Felder komplett neu importieren: 260$a, 260$b, 260$c, 300$a, 300$c, 593$a, 593$b, 592$a
- Dabei werden Felder manchmal komplett gelöscht, z.B. in https://muscat.rism.info/admin/sources/850634621 soll 260$a komplett verschwinden, wobei 260$c bleibt; bei https://muscat.rism.info/admin/sources/850726855 wird nichts in 260 vorhanden sein.
2. Die Bemerkungen im Feld 999 als neue(n) Bemerkungen ins Feld 500 verschieben. Das Feld 999 ist manchmal mehrmals vorhanden.
Beispiel: https://muscat.rism.info/admin/sources/850809583 hat zwei neue Bemerkungen: `Partitura d'orchestra`  und  `ms. copia`

**Quellen in diesem Ordner: https://muscat.rism.info/admin/folders/982**
1. Bemerkungen, die mit `Subject heading:` beginnen, löschen
2.  Folgende interne Bemerkung in 599$a ergänzen: `ICCU-Import 2. Paket (September 2023)`

Nach dem Import können die Titel [mit dieser Tabelle](https://docs.google.com/spreadsheets/d/1O_816-V9n3_vrGqaNCR6iXTA_lwt1RJXxl_QMBBSa48/edit#gid=1342152747) verglichen werden.

**Comment by HirschSt** [comment]
erledigt mit https://github.com/rism-international/muscat-maintenance/commit/7da4eb8b0ed89eda42cb37bd0868c098bf532a5f und https://github.com/rism-international/muscat-maintenance/commit/688de1370ea4e37f3bee91582780e6a2f67a7bf0

**Comment by jenniferward** [comment]
Die Wasserzeichen sind zwar im 592 gelandet, aber nicht im Materialblock?
https://muscat.rism.info/admin/sources/850699077/edit
"Ungrouped fields"
`=592  \\$aFiligrana: sigla V G,scudo a tre stelle$801` 

und hier ein Beispiel in 300:
https://muscat.rism.info/admin/sources/850726855/edit
`=300  \\$ascore$801`

Kann man generell nach "Nicht zugeteilten Feldern" suchen und entsprechend verschieben?

**Comment by HirschSt** [comment]
Ja, die Materialschicht wurde nicht ergänzt, das kann aber nachgeholt werden. Ich erstelle dafür ein eigenes Skript.

**Comment by jenniferward** [comment]
Neues Ticket: https://github.com/rism-international/tasks/issues/1021
